src/Entity/Video.php

```diff
@@ -15,6 +15,30 @@ class Video extends AbstractEntity
         'thumbnails' => 'YoutubeDl\\Entity\Thumbnail',
     ];
 
+    /**
+     * @return string
+     */
+    public function getAlbum()
+    {
+        return $this->get('album');
+    }
+
+    /**
+     * @return string
+     */
+    public function getArtist()
+    {
+        return $this->get('artist');
+    }
+
+    /**
+     * @return string
+     */
+    public function getTrack()
+    {
+        return $this->get('track');
+    }
+
```

Comment:
Are these columns always set? For example if using another provider than youtube? Could they return null?

Reply:
Youtube media tags are not always available, so yeah - they can return null. 

> The description of a video that contains music may include information about the song(s) in the video. YouTube adds this information automatically to some videos when the video has been identified and “claimed” by music rightsholders  via YouTube’s rights management tools, including Content ID.

Reply:
Could you update return typehints then? :)

Reply:
sure. commit 860bc57d14ad0f68b23e3b478804011a40441766